It's deadlocked here. I'm just removing these deprecated models, but the reason CI is failing is that these models can't run. Could you adjust the test cases? @crazywoola  Thank you.

```
=========================== short test summary info ============================
FAILED tests/test_llm_call.py::test_llm_invoke[x-ai/grok-4.1-fast] - ValueError: {'error': '{"args":{"description":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"Grok 4.1 Fast is deprecated. xAI recommends switching to Grok 4.3 (https://openrouter.ai/x-ai/grok-4.3)\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"},"error_type":"InvokeError","message":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"Grok 4.1 Fast is deprecated. xAI recommends switching to Grok 4.3 (https://openrouter.ai/x-ai/grok-4.3)\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"}'}
FAILED tests/test_llm_call.py::test_llm_invoke[x-ai/grok-4-fast] - ValueError: {'error': '{"args":{"description":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"Grok 4 Fast is deprecated. xAI recommends switching to Grok 4.3 (https://openrouter.ai/x-ai/grok-4.3)\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"},"error_type":"InvokeError","message":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"Grok 4 Fast is deprecated. xAI recommends switching to Grok 4.3 (https://openrouter.ai/x-ai/grok-4.3)\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"}'}
FAILED tests/test_llm_call.py::test_llm_invoke[x-ai/grok-3-beta] - ValueError: {'error': '{"args":{"description":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"No endpoints found for x-ai/grok-3-beta.\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"},"error_type":"InvokeError","message":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"No endpoints found for x-ai/grok-3-beta.\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"}'}
FAILED tests/test_llm_call.py::test_llm_invoke[x-ai/grok-3-mini-beta] - ValueError: {'error': '{"args":{"description":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"No endpoints found for x-ai/grok-3-mini-beta.\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"},"error_type":"InvokeError","message":"[models] Error: API request failed with status code 404: {\\"error\\":{\\"message\\":\\"No endpoints found for x-ai/grok-3-mini-beta.\\",\\"code\\":404},\\"user_id\\":\\"user_36KCsCfubXjVEK4OAW2cvVejzWn\\"}"}'}
```

--- a/models/openrouter/models/llm/_position.yaml
+++ b/models/openrouter/models/llm/_position.yaml
@@ -56,13 +56,12 @@
 - deepseek/deepseek-r1-distill-llama-70b
 - deepseek/deepseek-r1-distill-qwen-32b
 - moonshotai/kimi-k2-0905
+- xiaomi/mimo-v2.5-pro
+- xiaomi/mimo-v2.5
 - xiaomi/mimo-v2-omni
 - xiaomi/mimo-v2-pro
+- x-ai/grok-4.3
 - x-ai/grok-4.20
 - x-ai/grok-4.20-multi-agent
-- x-ai/grok-4.1-fast
-- x-ai/grok-4-fast
-- x-ai/grok-3-beta
-- x-ai/grok-3-mini-beta
 - z-ai/glm-5.1
 - z-ai/glm-5v-turbo

--- a/models/openrouter/models/llm/grok-4.3.yaml
+++ b/models/openrouter/models/llm/grok-4.3.yaml
@@ -1,0 +1,66 @@
+model: x-ai/grok-4.3
+label:
+  en_US: Grok 4.3
+model_type: llm
+features:
+  - multi-tool-call
+  - agent-thought
+  - stream-tool-call
+  - vision
+model_properties:
+  mode: chat
+  context_size: 1048576
+parameter_rules:
+  - name: response_format
+    label:
+      zh_Hans: 回复格式
+      en_US: Response Format
+    type: string
+    help:
+      zh_Hans: 指定模型必须输出的格式
+      en_US: specifying the format that the model must output
+    required: false
+    options:
+      - text
+      - json_object
+      - json_schema
+  - name: json_schema
+    use_template: json_schema
+  - name: enable_thinking
+    label:
+      zh_Hans: 开启思考模式
+      en_US: Enable thinking mode
+    type: boolean
+    default: true
+    help:
+      zh_Hans: 是否开启默认思考模式
+      en_US: Whether to enable the default thinking mode
+    required: false
+  - name: reasoning_effort
+    label:
+      zh_Hans: 推理程度
+      en_US: Reasoning Effort
+    type: string
+    help:
+      zh_Hans: 指定模型推理的难度
+      en_US: specifying the difficulty of the model's reasoning
+    required: false
+    options:
+      - high
+      - medium
+      - low
+  - name: exclude_reasoning_tokens
+    label:
+      zh_Hans: 隐藏思考过程
+      en_US: Hide the thought process
+    type: boolean
+    default: true
+    help:
+      zh_Hans: 是否隐藏思考过程。
+      en_US: Whether to hide the thought process.
+    required: false
+pricing:
+  input: "1.25"
+  output: "2.5"
+  unit: "0.000001"
+  currency: USD

--- a/models/openrouter/models/llm/mimo-v2.5-pro.yaml
+++ b/models/openrouter/models/llm/mimo-v2.5-pro.yaml
@@ -1,0 +1,29 @@
+model: xiaomi/mimo-v2.5-pro
+label:
+  en_US: MiMo-V2.5-Pro
+model_type: llm
+features:
+  - multi-tool-call
+  - agent-thought
+  - stream-tool-call
+model_properties:
+  mode: chat
+  context_size: 1048576
+parameter_rules:
+  - name: response_format
+    label:
+      zh_Hans: 回复格式
+      en_US: Response Format
+    type: string
+    help:
+      zh_Hans: 指定模型必须输出的格式
+      en_US: specifying the format that the model must output
+    required: false
+    options:
+      - text
+      - json_object
+pricing:
+  input: "1.00"
+  output: "3.00"
+  unit: "0.000001"
+  currency: USD

--- a/models/openrouter/models/llm/mimo-v2.5.yaml
+++ b/models/openrouter/models/llm/mimo-v2.5.yaml
@@ -1,0 +1,32 @@
+model: xiaomi/mimo-v2.5
+label:
+  en_US: MiMo-V2.5
+model_type: llm
+features:
+  - multi-tool-call
+  - agent-thought
+  - stream-tool-call
+  - vision
+  - audio
+  - video
+model_properties:
+  mode: chat
+  context_size: 1048576
+parameter_rules:
+  - name: response_format
+    label:
+      zh_Hans: 回复格式
+      en_US: Response Format
+    type: string
+    help:
+      zh_Hans: 指定模型必须输出的格式
+      en_US: specifying the format that the model must output
+    required: false
+    options:
+      - text
+      - json_object
+pricing:
+  input: "0.4"
+  output: "2.00"
+  unit: "0.000001"
+  currency: USD

